### ISSUE TITLE
I can't make it rotate

### ISSUE THREAD

**Issue by gersooonn** [bnsantoso/sub-to-audio#8]
Even though it seems so obvious, but I can't get it to run, and I've been after it for days. Turn .srt subtitles into audio
Could you help me make it run in Portuguese Brazil?

**Comment by bnsantoso** [comment]
I assume you want to use the fairseq model. The code below will ensure that the audio speed and position match the subtitle's duration and position. If it doesn't help, please tell me more about your problem and make sure all the dependencies are already installed on your computer.

```python
from subtoaudio import SubToAudio

sub = SubToAudio(language='por')
subtitle = sub.subtitle("yoursubtitle.srt")
sub.convert_to_audio(sub_data=subtitle, tempo_mode="precise")
```

Or you can try it in **Colab Notebook :** [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/bnsantoso/sub-to-audio//blob/main/subtitle_to_audio.ipynb). 
Fill the form like this `fairseq_language:"por"` , `subtitle_path:"yoursubtitle.srt"`, ` tempo_mode:"precise"` and run it one by one.

**Comment by gersooonn** [comment]
I managed to run it through the colab, it works with other coqui.tts models too, would it be the Bark model?
Do you have any means of contact, so that we can talk, I have some ideas, and another working model missing some adjustments for .srt, with you very realistic. maybe you might be interested

**Comment by bnsantoso** [comment]
yes i will fix the bark problem. well you can actualy write your idea here. :D
